src/components/30-organisms/testimonials/README.md

```diff
@@ -74,12 +68,9 @@ Import the testimonials-defining script and use a testimonials like this:
   </head>
   <body>
     <axa-testimonials
-      autorotatetime="1000"
-      autorotatedisabled="false"
-      keysenabled="true"
-      showallinline="false"
       title="The main title"
       subtitle="This is the subtitle"
+      keysenabled=""
```

Comment:
Why empty string here?

Reply:
This is just a variation. All these possibilities are correct:

> Correct would be either <div hidden> or <div hidden=""> or <div hidden="hidden"> or case-insensitive and single quotes/unquoted variations of any of them.

Luca said: for html examples set `=""`and for react examples just the attribute.